packages/manager/modules/pci/src/constants.js

```diff
@@ -156,6 +156,7 @@ export const PCI_REDIRECT_URLS = {
     paymentMethodAdd:
       'https://www.ovh.com/manager/dedicated/index.html#/billing/payment/method/add',
     support: 'https://www.ovh.com/manager/dedicated/index.html#/support',
+    newSupportTicket: 'https://www.ovh.com/manager/dedicated/index.html#/support/tickets/new',
```

Comment:
I think there is a `RedirectionService` in `@ovh-ux/manager-core` that we could use so this URL is centralized
I'm not sure what is the best way to do but pci module shouldn't have to handle some support module knowledge